src/gui/mainwindow.cpp

```diff
@@ -173,6 +173,10 @@ MainWindow::MainWindow(IGUIApplication *app, WindowState initialState)
     m_ui->actionManageCookies->setIcon(UIThemeManager::instance()->getIcon(u"browser-cookies"_s, u"preferences-web-browser-cookies"_s));
     m_ui->menuLog->setIcon(UIThemeManager::instance()->getIcon(u"help-contents"_s));
     m_ui->actionCheckForUpdates->setIcon(UIThemeManager::instance()->getIcon(u"view-refresh"_s));
+    m_ui->actionRSSReader->setIcon(UIThemeManager::instance()->getIcon(u"rss"_s));
+    m_ui->actionShowFiltersSidebar->setIcon(UIThemeManager::instance()->getIcon(u"view-sidetree"_s));
+    m_ui->actionSpeedInTitleBar->setIcon(UIThemeManager::instance()->getIcon(u"speedometer"_s));
+    m_ui->actionSearchWidget->setIcon(UIThemeManager::instance()->getIcon(u"search"_s));
```

Comment:
IIRC, there are no such icons in qBittorrent sources.

Reply:
Ok, I've just tested this on windows & here are the results:

Before This PR:
checkboxes/icons are all aligned vertically
![Screenshot 2024-02-16 112209](https://github.com/qbittorrent/qBittorrent/assets/42386382/115ca95f-95e5-44f5-a0e7-c4ba011c336c)

With this PR:
Only the `speedometer` is added & it takes the place of the checkbox!
![Screenshot 2024-02-16 112243](https://github.com/qbittorrent/qBittorrent/assets/42386382/6e5dfb57-9798-43db-a9ad-eb54bb0bd335)

`speedometer` - clicked/enabled
![Screenshot 2024-02-16 112310](https://github.com/qbittorrent/qBittorrent/assets/42386382/193efe1c-8a78-485a-80f1-b4a0a7c716d0)

Reply:
@buckmelanoma why are the results different than yours ?